org.ekstep.questionunit.sequence-1.1/renderer/js/seq-template.js

```diff
@@ -6,6 +6,7 @@ var SEQController = SEQController || {};
  */
 SEQController.initTemplate = function (pluginInstance) {
   SEQController.pluginInstance = pluginInstance;
+  SEQController.pluginInstance._attempted = false;
```

Comment:
Why dont you define this variable in the questionUnit & any question plugin can update the value based on the condition  specific to question plugin.

Reply:
So that it will work for all different versions of the plugins.